Samples/auth/Office-Add-in-ASPNET-SSO/README.md

```diff
@@ -100,4 +100,6 @@ Copyright (c) 2019 - 2021 Microsoft Corporation. All rights reserved.
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+**Note:** The Index.cshtml file contains an image URL that tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project.
```

Comment:
Nit: Colon should not also be formatted per [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28750). Here and other instances
```suggestion
**Note**: The Index.cshtml file contains an image URL that tracks diagnostic data for this sample add-in. Please remove the image tag if you reuse this sample in your own code project.
```